src/content/docs/en/reference/api-reference.mdx

```diff
@@ -238,7 +238,7 @@ Sets the cookie `key` to the given value. This will attempt to convert the cooki
 **Type:** `(key: string, options?: CookieDeleteOptions) => void`
 </p>
 
-Marks the cookie as deleted. Once a cookie is deleted `Astro.cookies.has()` will return `false` and `Astro.cookies.get()` will return an [`AstroCookie`](#astrocookie) with a `value` of `undefined`. Options available when deleting a cookie are: `domain`, `path`, `httpOnly`, `sameSite`, and `secure`.
+Marks the cookie as deleted by setting the expiration date to 0 in Unix time. Once a cookie is deleted `Astro.cookies.has()` will return `false` and `Astro.cookies.get()` will return an [`AstroCookie`](#astrocookie) with a `value` of `undefined`. Options available when deleting a cookie are: `domain`, `path`, `httpOnly`, `sameSite`, and `secure`.
```

Comment:
```suggestion
Invalidates a cookie by setting the expiration date in the past (0 in Unix time).

Once a cookie is "deleted" (expired), `Astro.cookies.has()` will return `false` and `Astro.cookies.get()` will return an [`AstroCookie`](#astrocookie) with a `value` of `undefined`. Options available when deleting a cookie are: `domain`, `path`, `httpOnly`, `sameSite`, and `secure`.
```
Great call out, @whitep4nth3r ! In fact, I like your plain language explanation in your PR description and blog post, so I think it's great to include it here so people have another way to parse what "0 in Unix time" means.

What do you think about my suggestion above?

Reply:
This is great, your suggestion gives more context than mine! 👍🏼